static/options.ts

```diff
@@ -22,18 +22,18 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-'use strict';
-
-var configElement = document.getElementById('config');
+const configElement = document.getElementById('config');
 
 window.httpRoot = configElement.getAttribute('httpRoot');
 window.staticRoot = configElement.getAttribute('staticRoot');
 
-var extraOptions = JSON.parse(decodeURIComponent(configElement.getAttribute('extraOptions')));
-for (var k in extraOptions) {
-    window.compilerExplorerOptions[k] = extraOptions[k];
+const extraOptions: object = JSON.parse(decodeURIComponent(configElement.getAttribute('extraOptions')));
+for (const key in extraOptions) {
+    window.compilerExplorerOptions[key] = extraOptions[key];
 }
 
-__webpack_public_path__ = window.staticRoot;
+declare var __webpack_public_path__: string;
+
+__webpack_public_path__ = window.httpRoot;
```

Comment:
Why did this change from `window.staticRoot` to `window.httpRoot`?
In production `staticRoot` will point to our CDN.

Reply:
Ups, that was not intended! Fixed.

Reply:
Ups, that was not intended! Fixed